orbit/pkg/table/windowsupdatetable/windowsupdate.go

```diff
@@ -27,6 +27,13 @@ const (
 	HistoryTable
 )
 
+type WindowsUpdatesSearcher interface {
```

Comment:
No need to export interface AFAICS.

Reply:
Done in 2499c119bb5b86a4d838971449cdd87b4fa6fcf0